gradle/versions.gradle

```diff
@@ -71,6 +71,7 @@ dependencyManagement {
     dependency 'org.apache.tuweni:tuweni-toml:0.10.0'
     dependency 'org.apache.tuweni:tuweni-units:0.10.0'
     dependency 'org.apache.tuweni:tuweni-net:0.10.0'
+    dependency 'org.apache.tuweni:tuweni-ethstats:1.0.0-SNAPSHOT'
```

Comment:
We cant merge this with a snapshot dependency.

Reply:
Yes, I am refraining to cut 1.0.0 quite yet until you review. Chicken and egg issue.